javascript/grid-ui/src/App.tsx

```diff
@@ -21,6 +22,7 @@ import NodeType from "./models/node";
 const cache = new InMemoryCache();
 const link = new HttpLink({
 	uri: GridConfig.serverUri,
+  fetch
```

Comment:
Is this line needed? It feels like this would error since it's not in an object format

Reply:
Yes, its needed.

There are some tests that fail with the following message:

```
Test suite failed to run

    Invariant Violation:
    fetch is not found globally and no fetcher passed, to fix pass a fetch for
    your environment like https://www.npmjs.com/package/unfetch.

    For example:
    import fetch from 'unfetch';
    import { createHttpLink } from 'apollo-link-http';

    const link = createHttpLink({ uri: '/graphql', fetch: fetch });

      20 |
      21 | const cache = new InMemoryCache();
    > 22 | const link = new HttpLink({
         |              ^
      23 | 	uri: GridConfig.serverUri,
      24 | });
      25 |

      at new InvariantError (../../../../../../../../../node_modules/ts-invariant/lib/invariant.esm.js:12:28)
      at Object.<anonymous>.exports.checkFetcher (../../../../../../../../../node_modules/apollo-link-http-common/src/index.ts:178:11)
      at Object.<anonymous>.exports.createHttpLink (../../../../../../../../../node_modules/apollo-link-http/src/httpLink.ts:46:3)
      at new HttpLink (../../../../../../../../../node_modules/apollo-link-http/src/httpLink.ts:259:11)
      at Object.<anonymous> (../../../../src/App.tsx:22:14)
      at Object.<anonymous> (../../../../src/core/Status.ts:2:1)
      at Object.<anonymous> (../../../../src/screens/Console/Console.tsx:14:1)
      at Object.<anonymous> (../../../../src/screens/Console/Console.keybinds.ts:4:1)
      at Object.<anonymous> (../../../../src/components/KeyBoard/KeyBoardHelp.tsx:7:1)
      at Object.<anonymous> (../../../../src/components/NavBar/NavBar.tsx:12:1)
      at Object.<anonymous> (../../../../src/tests/components/NavBar.test.tsx:3:1)
```

When checked on SO, that was the fix that was suggested on multiple questions of similar error stack trace.

Reply:
I am able to build and start the server. UI page opens fine.

Reply:
Ok, can we have it as a key/value pair then